### PR TITLE
Revives "Updates ERT gear and adds holo stretcher #16266"

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
This PR brings back the changes in https://github.com/ParadiseSS13/Paradise/pull/16266, as they were good, but due to factors unrelated to the PR it was closed early.

* Moves all ERT firearms from their right hand to their left.
* Moves T4 charges and mindshields for ERT from lockboxes to cardboard boxes.
* Gives the Red and Gamma ERT medics a bottle of painkillers.
* Makes ERT PDAs detonate immune and gives them the same functionality as the captain.
* Adds a holo stretcher, a roller bed that fits in a pocket. All levels of medical ERT start with one in lieu of a roller bed.
* Science can make a holo stretcher with biotech 5, magnets 4, and power systems 4. It takes 1000 metal, 500 silver, 500 glass, and 200 diamond.
* The surgery success chance of a roller bed was taken from 70% to 90%.
* Roller bed moved back to bulky size now that there is an upgrade path to holo stretcher

## Why It's Good For The Game

* A large number of friendly fire incidents are caused by the guns being selected when the ERT spawns in.
* It's very frustrating that ERT starts with items in their backpacks that don't fit in a backpack, and it's not like the lockbox is going to keep anyone out since killing the ERT will give you an ID that will open the lockbox.
* Doing surgery without painkillers or anesthetics is very difficult.
* The fact that ERT has PDAs on par with civilians is just sad, and it provides them a useful set of tools.
* Like with the lockboxes, this exists to remove an item from ERT that won't fit back in their backpacks.
* The item seemed useful and had a very lovely sprite courtesy of Stokes52, so I wanted the item to see more use. It was balanced to be cheap to make but requires science to have gotten materials from mining before it can be made.
* Currently, a medical bed has a lower surgery success chance than a kitchen table, as it shares a 70% success rate with all other beds. This keeps it inferior to operation tables but makes them the next best option for operating. Further, it makes it practical, even if not ideal for ERT to do field surgery.

## Images of changes

image
image

## Changelog
:cl: S34N, warior4356
add: Added holo stretcher to ERT loadouts and science, holo stretchers have a 90% surgery success rate
add: Added a bottle of painkillers to Red and Gamma Medical ERTs
add: Added all tools and detonate immunity to ERT PDAs
tweak: Regular roller beds are bulky again, holostretchers are small
tweak: Moved ERT T4 and mindshields from lockboxes to cardboard
tweak: Moved ERT guns from their right hard to their left
tweak: Increased surgery success rate for roller beds to 80%
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -880,6 +880,24 @@
...
+		new /obj/item/grenade/plastic/c4/thermite(src)
+
+/obj/item/storage/box/mindshield
+	name = "boxed mindshield kit"
+	desc = "Contains everything needed to secure the minds of those around you."
+
+/obj/item/storage/box/mindshield/populate_contents()
+	for(var/I in 1 to 3)
+		new /obj/item/implantcase/mindshield(src)
+	new /obj/item/implanter/mindshield(src)
```
Comment (moxian) [inline code/game/objects/items/weapons/storage/boxes.dm:899]: why not just `for(var/I in 1 to 4)` ?..

File: code/modules/pda/pdas.dm [hunk 2]
```diff
@@ -90,6 +90,8 @@
 	icon_state = "pda-h"
 
 /obj/item/pda/heads/ert
+	default_cartridge = /obj/item/cartridge/centcom
+	detonate = FALSE
```
Comment (moxian) [inline code/modules/pda/pdas.dm:94]: While you're here, mind changing the definition in `/obj/item/pda` to `var/detonate = TRUE`, and the (only other) override in `/obj/item/pda/silicon` to `detonate = FALSE` ?

File: code/modules/research/designs/medical_designs.dm [hunk 3]
```diff
@@ -62,6 +62,16 @@
...
 	category = list("Medical")
 
+/datum/design/holostretcher
+	name = "Holo Strecher"
+	desc = "A hardlight projector for transporting patients."
+	id = "holo_stretcher"
+	req_tech = list("biotech" = 5, "magnets" = 4, "powerstorage" = 4)
+	build_path = /obj/item/roller/holo
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 1000, MAT_SILVER = 500, MAT_GLASS = 500, MAT_DIAMOND = 200)
```
Comment (moxian) [inline code/modules/research/designs/medical_designs.dm:72]: Needing diamond might be a bit too expensive?
I don't expect people to want to print these anyway, making it cost lots of resources is probably not needed.

But then again, i'm not a science player, so i'm not quite sure if 200 diamond is "a lot"...

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (S34NW) [reply code/game/objects/items/weapons/storage/boxes.dm:899]: look closely

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 4]
```diff
@@ -74,6 +74,9 @@
 	resistance_flags = NONE
 	anchored = FALSE
 	comfort = 1
+	var/up_state = "up"
+	var/down_state = "down"
```
[the remark this answers is not in the document]
Comment (S34NW) [reply code/game/objects/structures/stool_bed_chair_nest/bed.dm:78]: sure

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (moxian) [reply code/game/objects/items/weapons/storage/boxes.dm:899]: derp

File: code/modules/research/designs/medical_designs.dm (hunk 3 above)
Comment (S34NW) [reply code/modules/research/designs/medical_designs.dm:72]: 200 is not a lot, and upgrades reduce this further

**Comment by S34NW** [comment]
well that went about as well as expected for a webedit, one mo

File: code/modules/surgery/helpers.dm [hunk 5]
```diff
@@ -147,12 +147,13 @@
 	var/turf/T = get_turf(M)
 	if(locate(/obj/machinery/optable, T))
 		return 1
-	else if(locate(/obj/structure/table, T))
+	if(locate(/obj/structure/bed/roller/holo, T))
+		return 0.9
+	if(locate(/obj/structure/table, T) || locate(/obj/structure/bed/roller, T))
 		return 0.8
```
Comment (alex-gh) [inline code/modules/surgery/helpers.dm:153]: Would it make sense balance wise to bump normal roller beds to 0.9 as well? 
After all tables are ubiquitous, while roller beds are a lot rarer.

Comment (S34NW) [reply code/modules/surgery/helpers.dm:153]: This was the order one of the balance team members requested. I'm not opposed to changing it but I'd need some of them to ask or be okay with it

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 6]
```diff
@@ -92,30 +95,41 @@
...
+	M.pixel_y = initial(M.pixel_y)
 	..()
 
 /obj/structure/bed/roller/post_unbuckle_mob(mob/living/M)
 	density = FALSE
-	icon_state = "down"
+	icon_state = icon_down
+	M.pixel_x = M.get_standard_pixel_x_offset()
+	M.pixel_y = M.get_standard_pixel_y_offset()
 	..()
```
Comment (hal9000PR) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:114]: this isn't needed as ..() is called

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 7]
```diff
@@ -92,30 +95,41 @@
...
 			qdel(src)
 	else
 		return ..()
 
 /obj/structure/bed/roller/post_buckle_mob(mob/living/M)
 	density = TRUE
-	icon_state = "up"
+	icon_state = icon_up
+	M.pixel_y = initial(M.pixel_y)
 	..()
```
Comment (hal9000PR) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:107]: same here

File: code/modules/research/designs/medical_designs.dm [hunk 8]
```diff
@@ -62,6 +62,16 @@
 	build_path = /obj/item/handheld_defibrillator
 	category = list("Medical")
 
+/datum/design/holostretcher
+	name = "Holo Strecher"
+	desc = "A hardlight projector for transporting patients."
+	id = "holo_stretcher"
+	req_tech = list("biotech" = 5, "magnets" = 4, "powerstorage" = 4)
```
Comment (hal9000PR) [inline code/modules/research/designs/medical_designs.dm:69]: biotech feels weird for this, it involves zero biology? maybe replace it by upping the magnets to 5? 6?

**Review by hal9000PR — DISMISSED** [review]

File: code/modules/pda/pdas.dm (hunk 2 above)
Comment (farie82) [inline code/modules/pda/pdas.dm:94]: Can you add this to the CL?

File: code/modules/surgery/helpers.dm (hunk 5 above)
Comment (farie82) [inline code/modules/surgery/helpers.dm:153]: Can you update the CL to show that roller beds have a 80% chance while holo roller beds have a 90% chance?

**Review by farie82 — APPROVED** [review]
Code looks good. Just a few CL related things. I'll merge once those are changed

File: code/modules/pda/pdas.dm (hunk 2 above)
Comment (S34NW) [reply code/modules/pda/pdas.dm:94]: This is in the CL already :)

File: code/modules/surgery/helpers.dm (hunk 5 above)
Comment (S34NW) [reply code/modules/surgery/helpers.dm:153]: I'll add this now

**Pull request merged**